packages/api-client/src/components/CommandPalette/CommandPaletteRequest.vue

```diff
@@ -103,6 +126,24 @@ onMounted(() => {
           :method="requestMethod"
           @change="handleChangeMethod" />
         <ScalarListbox
+          v-model="selectedCollection"
+          :options="collections">
+          <ScalarButton
+            class="justify-between p-2 max-h-8 w-full gap-1 text-xs hover:bg-b-2"
+            variant="outlined">
+            <span :class="selectedCollection ? 'text-c-1' : 'text-c-3'">{{
+              selectedCollection
+                ? selectedCollection.label
+                : 'Select Collection'
```

Comment:
```suggestion
                selectedCollection?.label ?? 'Select Collection'
```
we love nullish coalescing